.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we add all eclipse specific files to gitignore?

Reply:
i think it is best to add the IDE specific files, it allows a faster setup of the project. These files can be filtered in other IDEs.

I really this talk explaining why ;-)
https://www.youtube.com/watch?v=J5RzSbpNLM4&list=PLU-T8l-XOWOMOVW9RrR9Vw4kc8fGzGpd1&index=4&t=2s

Reply:
ok. yeah, we typically assume only maven based builds so my comment was coming from there. But your argument is valid.